jupyter/Caddyfile

```diff
@@ -0,0 +1,16 @@
+example.com {
+
+ 
+  proxy / localhost:8888 {
+    header_upstream X-Real-IP  {remote}
+    header_upstream Host {host}
+    websocket
+  }
+
+    rewrite {
+        r "~* /(api/kernels/[^/]+/(channels|iopub|shell|stdin)|terminals/websocket)/?"
```

Comment:
Have you tested this expression? I'm not familiar with the `~* ` prefix. What does it do?